src/test/java/com/wikia/webdriver/testcases/desktop/adstests/TestUrlBuilder.java

```diff
@@ -265,6 +265,7 @@
 
   @Test(groups = {"TestUrlBuilder", "unitTests"})
   public void urlBuilderTests() throws Exception {
+    Configuration.setTestValue("forceFandomDomain", "false");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "forceFandomDomain" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "false" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)